src/assessments/contrast/test-steps/state-changes.tsx

```diff
@@ -24,32 +24,29 @@ const howToTest: JSX.Element = (
         <p>The visual helper for this requirement highlights links, native widgets, and custom widgets in the target page.</p>
         <ol>
             <li>
-                In the target page, examine each highlighted element to determine whether it can adopt any of the following states:
+                In the target page, examine each highlighted element to determine whether it supports any of the following states:
                 <ol>
                     <li>Focused</li>
                     <li>Mouseover</li>
                     <li>Selected</li>
                 </ol>
             </li>
             <li>
-                Identify any visual (non-text) indicators that communicate:
+                In each supported state, use <WindowsContrastCheckerAppLink /> (or the <MacContrastCheckerAppLink /> if you are testing on a
+                Mac) to verify that the following visual presentations (if implemented) have a contrast ratio of at least 3:1 against the
+                adjacent background:
                 <ol>
-                    <li>The boundary of the component's clickable area (such as a border or background color)</li>
-                    <li>The component's current state (such as a different background color or a checkmark)</li>
+                    <li>Any visual boundary that indicates the component's clickable area</li>
```

Comment:
nit, bug from the spec: its missing a "." at the end

Reply:
updated